pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -65,7 +65,7 @@ import (
 const LibvirtLocalConnectionPort = 22222
 
 type DomainManager interface {
-	SyncVMI(*v1.VirtualMachineInstance, bool) (*api.DomainSpec, error)
+	SyncVMI(*v1.VirtualMachineInstance, bool, cmdclient.VirtualMachineOptions) (*api.DomainSpec, error)
```

Comment:
I think it'll be better to have options as a pointer

Reply:
Added